PROJECT_PROGRESSION.md

```diff
@@ -153,8 +153,12 @@ It is expected that incubating project will make an active effort to work throug
 
 **Acceptance Criteria**
 
+1. consensus within private mailing list to move into incubation process
+
+#### Graduation Criteria
```

Comment:
```suggestion
**Graduation Criteria**
```

All of the other sub-titles are using `**bold**` rather than `#### h4` styling. This should match the rest. Alternatively, the styling of the other titles should be fixed.

Reply:
Heading styling creates permalinks, so it’s a better choice.

Reply:
That’s been fixed for the rest of the page with https://github.com/openjs-foundation/cross-project-council/pull/293 which has already been merged.